patches/coreclr/0002-clang9-resolution-to-conditionally-include-allocator.patch

```diff
@@ -0,0 +1,27 @@
+From 2b5e1784bdd399e5c1112935546757f1b0542326 Mon Sep 17 00:00:00 2001
+From: Aditya Aggarwal <aditya.aggarwal@microsoft.com>
+Date: Fri, 27 Mar 2020 19:18:42 +0000
+Subject: [PATCH 2/2] clang9 resolution to conditionally include allocator
+ header file
```

Comment:
Include link to https://github.com/dotnet/source-build/issues/1449 in this patch file?

(I seem to remember we were wanting to include a link in every patch file so we could track removing them. Also makes it easier to figure out what's going on when reapplying them.)